tests/Functional/ContentNewsRenderTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\oe_theme\PatternAssertions\FieldListAssert;
+use Drupal\user\Entity\Role;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests that our News content type renders correctly.
+ */
+class ContentNewsRenderTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'oe_theme_helper',
+    'path',
+    'oe_theme_content_news',
+    'block',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Enable and set OpenEuropa Theme as default.
+    \Drupal::service('theme_installer')->install(['oe_theme']);
+    \Drupal::configFactory()->getEditable('system.theme')->set('default', 'oe_theme')->save();
+    // Rebuild the ui_pattern definitions to collect the ones provided by
+    // oe_theme itself.
+    \Drupal::service('plugin.manager.ui_patterns')->clearCachedDefinitions();
+
+    Role::load(RoleInterface::ANONYMOUS_ID)
+      ->grantPermission('view published skos concept entities')
+      ->save();
+  }
+
+  /**
+   * Tests that the Project page renders correctly.
+   */
+  public function testNewsRendering(): void {
+    // Create a Project node.
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = $this->getStorage('node')->create([
+      'type' => 'oe_news',
+      'title' => 'Test news node',
+      'oe_teaser' => 'News teaser',
```

Comment:
lets also test rendering of oe_news_featured_media

Reply:
The aim of the ticket is to add Reference field and theme Details group.

Reply:
if it will be in a follow up then its ok

Reply:
Yeah we will merge the two tests later on, depending which issue gets merged first, this one or #583